README.md

```diff
@@ -52,17 +52,17 @@ Options:
   --normalize <number>         normalize icons by scaling them to the height of the highest icon
   -r, --round [bool]           setup the SVG path rounding [10e12]
   --selector <value>           use a CSS selector instead of 'tag + prefix' (default: null)
-  -, --prefix <value>          CSS class prefix (default: icon)
+  -p, --prefix <value>         CSS class prefix (default: icon)
   --tag <value>                CSS base tag for icons (default: i)
-  -u, --fonts-url <value>      public url to the fonts directory (used in the generated CSS)
+  -u, --fonts-url <value>      public URL to the fonts directory (used in the generated CSS)
```

Comment:
This is good but we should have it in the `cli/index.ts` itself (this block of options is a copy paste I've been keeping up to date of the `fantasticon --help` output

Reply:
(Of course happy to merge first now and fix that later)

Reply:
Pushed